### PR TITLE
Fix/npm dependencies

### LINKED ISSUE #105: BUG: skeleton not working when installing from scratch
There is probably a module dependency missing (glob) and with some node version the npm scripts are not running (seen on snap-ci).

### PR DESCRIPTION
**Pull request by lejoe**
Fixes #105

### CODE REVIEW

**Comment by lejoe** [comment]
Had to setup snap-ci again but it should work now.
The run goes through: https://snap-ci.com/ginetta/skeleton/branch/fix/npm-dependencies

**Comment by meodai** [comment]
worked for me!

File: package.json [hunk 1]
```diff
@@ -4,11 +4,12 @@
...
-    "start": "./node_modules/.bin/gulp",
-    "build": "./node_modules/.bin/gulp build",
-    "test": "./node_modules/.bin/gulp test",
-    "serve": "./node_modules/.bin/gulp serve",
-    "deploy": "./node_modules/.bin/gulp deploy"
+    "gulp": "gulp",
+    "start": "npm run gulp",
+    "build": "npm run gulp -- build",
+    "test": "npm run gulp -- test",
+    "serve": "npm run gulp -- serve",
```
Comment (peterbraden) [inline package.json]: Why not just use 'gulp serve' etc for these?

Comment (lejoe) [inline package.json]: Right, I misread http://stackoverflow.com/questions/22224831/no-command-gulp-found-after-installation/22274098#22274098 and http://engineering.hobsons.com/2015/06/26/build-tools-vs-npm-scripts-why-not-both/

Will do a fix for this.

**Pull request merged**